### DISCUSSION TITLE
FR: Control Panel - Section Description

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8689] [Ideas]
For content heavy sites with lots of defined sections it would be great if Craft provided an ability for optional text describing each section to make it easier for content authors to understand what type of content each sections holds.

When creating a new section within the control panel it would be nice to have an optional built-in "description" field so a short summary describing what content the section holds and any other relevant high-level information could be added.

When navigating the entry index it would then be neat if these short descriptions could be displayed at the top of each section entry listing (maybe an "I" icon which when clicked displays a modal with the description text). Another idea would be to display the "I" icon to the left of the section name in the left section navigation as shown in the screenshot below.

![sections](https://user-images.githubusercontent.com/2505643/58522389-7cb10700-8203-11e9-827b-feb3081c399c.png)

**Comment by piotrpog** [comment]
Good idea! I would not need to create readme files for my users.

**Comment by KatieMFritz** [comment]
I've wished for this too!
